sympy/physics/quantum/tests/test_operator.py

```diff
@@ -231,3 +231,26 @@ def test_differential_operator():
     assert diff(d, th) == \
         DifferentialOperator(Derivative(d.expr, th), f(r, th))
     assert qapply(d*w) == Wavefunction(3*sin(th), r, (th, 0, pi))
+
+
+def test_eval_power():
+    from sympy import Pow
+    O = Operator('O')
+    U = UnitaryOperator('U')
+    H = HermitianOperator('H')
+    assert O**-1 == O.inv() # same as doc test
+    assert U**-1 == U.inv()
+    assert H**-1 == H.inv()
+    x = symbols("x", commutative = True)
+    assert H**x == Pow(H, x, evaluate=False)
```

Comment:
There is a function `unchanged` in `sympy/core/expr.py` for this sort of test.

Reply:
Good to know. However in this case `assert unchanged(Pow, H, x)` doesn't seem to be clearer in any way than `Pow(H, x,  evaluate=False)`.

Reply:
It's not so much clearer as more correct. The test above would pass even if `H**x` evaluated as something completely different and `Pow` ignored the `evaluate=False` argument. You're asserting equality of two things but that is not the same as asserting that either of the two things is what they should be.

Reply:
Correct. I have added several tests:
1) Initially I only wanted to verify that there is no bug in the interpretation of `**`, as SymPy doesn't guarantee `H**x == Pow(H, x)`.  So `H**x` should evaluate to `T` where `T` is the SymPy expr that represents the mathematical term `H to the x` (not its value).  I had relied on `T == Pow(H, x, evaluate=False)`.
Since `unchanged` needs a function argument, this test cannot be formulated using `unchanged` .
2) However I should better also verify Pow(H,x)==T. This is best done using `assert unchanged(Pow, H, x)` 
3) So with assertion 2 in place, Test 1 ist shortest written `assert H**x == Pow(H, x)` 
4) You pointed out that `Pow(H, x, evaluate=False) == T` could not be relied upon here. In my eyes that would indicate a serious bug. So I added an `assert Pow(H,x,evaluate=False) == Pow(H,x)`